site/app/templates/admin/LateDays.twig

```diff
@@ -77,4 +77,34 @@
     $("#user_id").autocomplete({
         source: {{ student_full|raw }}
     });
+    flatpickr("#datestamp", {
+            plugins: [ShortcutButtonsPlugin(
+                    {
+                        button: [
+                            {
+                                label: "Now"
+                            }, 
+                            {
+                                label: "End of time"
+                            }
+                        ],
+                        label: "or",
+                        onClick: (index, fp) => {
+                            let date;
+                            switch (index) {
+                                case 0: 
+                                    date = new Date();
+                                    break;
+                                case 1: 
+                                    date = new Date("01/01/98");
```

Comment:
I noticed that `01/01/98` seems to be interpreted as `1998-01-01` in the date picker. Should be 9998 though.

That being said, consider https://github.com/Submitty/Submitty/issues/3402, it looks the year will be set to 1998 anyways. @bmcutler do we want to use four-digit year in the late days page to address #3402 in another PR?

Reply:
Ah! yeah... Looks like we would have to modify some operations done on the date should we change the format, let's hear what Barb has to say.